tests/client/test_dataset_version_export.py

```diff
@@ -0,0 +1,128 @@
+                            ),
+                        ),
+                        dataset_ids=[dataset_id],
+                    ),
+                ],
+            ),
+            metadata=metadata(),
+        )
+        raise_on_failure(post_inputs_response)
+
+        input_ids = [input.id for input in post_inputs_response.inputs]
+        wait_for_inputs_upload(stub, metadata(), input_ids)
+
+        dataset_version_id = dataset_id  # Reuse dataset external ID for version.
+        post_dataset_versions_response = stub.PostDatasetVersions(
+            service_pb2.PostDatasetVersionsRequest(
+                dataset_id=dataset_id,
+                dataset_versions=[resources_pb2.DatasetVersion(id=dataset_version_id)],
+            ),
+            metadata=metadata(),
+        )
+        raise_on_failure(post_dataset_versions_response)
+
+        wait_for_dataset_version_ready(stub, metadata(), dataset_id, dataset_version_id)
+
+        put_dataset_version_exports_response = stub.PutDatasetVersionExports(
+            service_pb2.PutDatasetVersionExportsRequest(
+                dataset_id=dataset_id,
+                dataset_version_id=dataset_version_id,
+                exports=[
+                    resources_pb2.DatasetVersionExport(
+                        format=resources_pb2.CLARIFAI_DATA_PROTOBUF,
+                    ),
+                ],
+            ),
+            metadata=metadata(),
+        )
+        raise_on_failure(put_dataset_version_exports_response)
+
+        wait_for_dataset_version_export_success(stub, metadata(), dataset_id, dataset_version_id)
+
+        get_dataset_version_response = stub.GetDatasetVersion(
+            service_pb2.GetDatasetVersionRequest(
+                dataset_id=dataset_id,
+                dataset_version_id=dataset_version_id,
+            ),
+            metadata=metadata(),
+        )
+        raise_on_failure(get_dataset_version_response)
+
+        export = get_dataset_version_response.dataset_version.export_info.clarifai_data_protobuf
+        assert export.format == resources_pb2.CLARIFAI_DATA_PROTOBUF
+        assert export.size > 0
+
+        get_export_url_response = requests.get(export.url)
+        assert get_export_url_response.status_code == 200
+
+        with zipfile.ZipFile(io.BytesIO(get_export_url_response.content)) as zip_file:
+            assert zip_file.read("mimetype") == b"application/x.clarifai-data+protobuf"
+
+            namelist = zip_file.namelist()
+            namelist.remove("mimetype")
+            assert len(namelist) == 1  # All inputs in a single batch.
+
+            input_batch = resources_pb2.InputBatch().FromString(zip_file.read(namelist[0]))
+            assert len(input_batch.inputs) == len(input_ids)
+    finally:
+        delete_datasets_response = stub.DeleteDatasets(
+            service_pb2.DeleteDatasetsRequest(dataset_ids=[dataset_id]),
+            metadata=metadata(),
+        )
```

Comment:
As mentioned in the Jira ticket comments, this does not delete the dataset version, so do that explicitly.